github/Organization.py

```diff
@@ -675,6 +675,31 @@ def create_secret(
         )
         return status == 201
 
+    def add_repository_to_secret(
+        self,
+        secret_name,
+        repository_id,
+        secret_type,
+    ):
+        """
+        :calls: `PUT /orgs/{org}/actions/secrets/{secret_name}/repositories/{repository_id} <https://docs.github.com/en/rest/actions/secrets#add-selected-repository-to-an-organization-secret>`_
+                `PUT /orgs/{org}/dependabot/secrets/{secret_name}/repositories/{repository_id} <https://docs.github.com/en/rest/dependabot/secrets#add-selected-repository-to-an-organization-secret>`_
```

Comment:
Does this fail the docs build? It's actually /orgs/{org}/{secret_type} anyway.

Reply:
fixed please recheck